.circleci/config.yml

```diff
@@ -70,10 +75,25 @@ jobs:
             docker load -i ./docker-images/spring-cloud-kubernetes-discoveryserver.tar
             docker load -i ./docker-images/spring-cloud-kubernetes-configserver.tar
       - run:
-          name: Run Kind Integration Tests
+          name: testcontainers integration tests
```

Comment:
Do you think it would be possible to run the integration tests as part of the regular build?  So ultimately we could just run `./mvnw clean install` and everything gets run?

Reply:
YES! initially I did not believe this is feasible, but took my time (cause I really liked this idea of yours) and it's possible.